### DISCUSSION TITLE
LogsActivity Trait not logging when used in model

### DISCUSSION THREAD

**Discussion by HamzaHummam** [spatie/laravel-activitylog#1151] [Q&A]
I have used LogsActivity Trait and Implemented getActivitylogOptions() method, but create,update,delete events are not logged in activity_log table need help, Thank you

**Comment by parallels999** [comment]
https://spatie.be/docs/laravel-activitylog/v4/introduction

**Reply by HamzaHummam** [reply]
I have followed the documentation, manual event logging is working fine but I want to it to be auto through model using LogActivity Trait as suggested in documentation. I have followed each step mentioned in documentation but it is not working.
